src/project/project.controller.ts

```diff
@@ -220,14 +219,46 @@ export class ProjectController {
     );
   }
 
+  @UseGuards(...IsEmailVerifiedGuard)
+  @ApiCookieAuth()
+  @Put(':id/invitation/accept')
+  async acceptEnrollRequest(
+    @Req() request: RequestWithUser,
+    @Param('id', ParseIntPipe) id: number,
+    @Body() enrollRequestAdminDto: EnrollmentRequestDto,
+  ) {
+    await this.projectService.manageEnrollInvitation(
+      id,
+      request.currentUser,
+      enrollRequestAdminDto,
+      'accept',
+    );
+  }
+
+  @UseGuards(...IsEmailVerifiedGuard)
+  @ApiCookieAuth()
+  @Put(':id/invitation/decline')
+  async declineEnrollRequest(
```

Comment:
Same here, We might need to modify this method name to 'declineEnrollInvitation' in order to match what is being called from the service and better readability